app/src/main/AndroidManifest.xml

```diff
@@ -23,6 +23,14 @@
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
+        <activity android:name=".UIActivity" android:screenOrientation="landscape"
+            >
+            <intent-filter>
+                <action android:name="android.intent.action.MAIN" />
+
+                <category android:name="android.intent.category.LAUNCHER" />
```

Comment:
No, I added it there temporarily as suggested by Javi because at some point we need to perform a merging of the mainActivity and the UIActivity.
The UI is only landscape to optimise the design.